BunqSdk/Model/Generated/Endpoint/MasterCardAction.cs

```diff
@@ -195,6 +201,11 @@ public static BunqResponse<List<MasterCardAction>> List(ApiContext apiContext, i
         /// </summary>
         public override bool IsAllFieldNull()
         {
+            if (this.Id != null)
```

Comment:
`this.Id.HasValue`?

Reply:
Generated code, please create follow up issue as this requires changes in the generator it self.